device.c

```diff
@@ -139,6 +139,7 @@ struct iio_channel * iio_device_find_channel(const struct iio_device *dev,
 			continue;
 
 		if (!strcmp(chn->id, name) ||
+				(chn->label && !strcmp(chn->label, name)) ||
```

Comment:
nothing wrong with the patch but commit subject does not match the style :)

Reply:
do you mean that there is an extra tab on this line?

Reply:
no.. the commit title itself. nothing wrong with the code. It's a nitpick but consistency across git log is a good thing. Also good for grepping the history

Reply:
Oh, I got it now. I need to grab another coffee.